playbooks/collect_activity/collect_activity.yml

```diff
@@ -22,11 +22,12 @@
       become_user: www-data
 
     - name: Run collect_activity script
-      command: >
+      shell: >
+        . /edx/app/edxapp/edxapp_env &&
```

Comment:
I think `source /edx/app/edxapp_env` would be more explicit here

Reply:
@pcockwell `shell` module uses `/bin/sh` under the hood, which has no `source` command.